tsconfig.json

```diff
@@ -7,12 +7,26 @@
     "emitDecoratorMetadata": true,
     "experimentalDecorators": true,
     "target": "es5",
-    "typeRoots": ["node_modules/@types"],
-    "lib": ["es2017", "dom"],
+    "typeRoots": [
+      "node_modules/@types"
+    ],
+    "lib": [
+      "es2017",
+      "dom"
+    ],
     "baseUrl": ".",
     "paths": {
-      "@orchestrator/core": ["libs/core/src/public_api.ts"]
+      "@orchestrator/core": ["libs/core/src/public_api.ts"],
+      "@orchestrator/layout": [
+        "libs/layout/src/public_api.ts"
+      ],
+      "@orchestrator/test-utils": [
```

Comment:
Can you please move this into secondary entry point in the `core` module under `testing` name as we discussed earlier?

Reply:
I tried to do this and it wasn't working properly. I will try again and let you know what's up.

Reply:
Were you able to complete this? I would like to merge your PR.